### PR TITLE
Added Gridlab-d package. 

### PR DESCRIPTION
**Pull request by 0t1s1**
Not including Matlab support at this time.

### CODE REVIEW

File: var/spack/repos/builtin/packages/gridlab-d/package.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class GridlabD(AutotoolsPackage):
+    """
+    Autotools package for Gridlab-D. One day the devs will convert to Cmake!
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/gridlab-d/package.py]: Can you change the docstring to be more descriptive?

**Review by alalazo — CHANGES_REQUESTED** [review]
A minor request, otherwise LGTM.

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
@yee29 @afisher1 If you're fine with being maintainers of the `gridlab-d` Spack package I'll merge this PR. This means that you'll be pinged for review when somebody will update this build recipe.

**Comment by yee29** [comment]
That is fine with me

**Review by alalazo — APPROVED** [review]

**Pull request merged**